admin/roles.sql

```diff
@@ -50,3 +50,7 @@ CREATE ROLE IF NOT EXISTS SAGE_CITATIONS_ANALYST;
 
 -- SAGE DB Admin
 CREATE ROLE IF NOT EXISTS SAGE_ADMIN;
+
+-- GOOGLE_ANALYTICS_AGGREGATE roles
+-- We already have a GOOGLE_ANALYTICS_AGGREGATE_ANALYST role (created by application)
```

Comment:
So this means this role `GOOGLE_ANALYTICS_AGGREGATE_ANALYST` is auto created elsewhere (also just maintained elsewhere) and we don't have to maintain here?

Reply:
I was just wondering in the scenario the role was manually dropped for example, but since only the Google Analytics app owns it and handles the grants and such there, not really for this end to handle it

Reply:
It's an account role, so while the installation of the Google Analytics app was the trigger which created it, it's owned by USERADMIN like all our other account roles. Check out the section of the Confluence doc on roles and privileges if you're curious how that all works: https://sagebionetworks.jira.com/wiki/spaces/DPE/pages/4383473698/Google+Analytics#Roles-and-Privileges